sgm/modules/diffusionmodules/lpips/model.py

```diff
@@ -0,0 +1,88 @@
+import functools
+
+import torch.nn as nn
+
+from .util import ActNorm
+
+
+def weights_init(m):
+    classname = m.__class__.__name__
+    if classname.find("Conv") != -1:
+        nn.init.normal_(m.weight.data, 0.0, 0.02)
+    elif classname.find("BatchNorm") != -1:
+        nn.init.normal_(m.weight.data, 1.0, 0.02)
+        nn.init.constant_(m.bias.data, 0)
+
+
+class NLayerDiscriminator(nn.Module):
+    """Defines a PatchGAN discriminator as in Pix2Pix
+    --> see https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/networks.py
```

Comment:
https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/LICENSE

Reply:
Done